### PR TITLE
Fix: Change i18n config and update translations

### PR DESCRIPTION
**Pull request by bbaa77770**

### CODE REVIEW

**Review by parachvte — CHANGES_REQUESTED** [review]
Keep up i18n key convention with @voidrin

**Comment by voidrin** [comment]
# About Resource Naming Convention

String keys can be naming by this following convention:

`<WHAT>_<WHERE>_<DESCRIPTION>`

As the string keys naming, I recommend to be connected by hyphen(not sure hyphen is permitted) or underscore instead of empty space.

### Advantages
- ordering the resource
- better readability and logically
- avoid naming conflict

### Disadvantages
- hard to refactor
- need to describe it clearly

## Examples
### Good
`laser_camera_aid_background_instructions_step1`

###  Not Good
`instructions of laser camera steps 1 1`
`instructions_laser_camera_4`

File: src/app/config/settings.js [hunk 1]
```diff
@@ -54,7 +54,8 @@ const settings = {
         load: 'currentOnly',
 
         // char to separate keys
-        keySeparator: '.',
+        // Todo: change a setting later
+        keySeparator: ' ',
```
Comment (parachvte) [inline src/app/config/settings.js:58]: false?

**Comment by parachvte** [comment]
LU-408 Updates i18n copy

**Comment by parachvte** [comment]
LU-408 Updates i18n copy

**Pull request merged**